src/app-layout/visual-refresh-toolbar/navigation/index.tsx

```diff
@@ -37,6 +37,7 @@ export function AppLayoutNavigationImplementation({ appLayoutInternals }: AppLay
         styles.navigation,
         {
           [styles['is-navigation-open']]: navigationOpen,
+          [testutilStyles['drawer-closed']]: !navigationOpen,
```

Comment:
Not related to deduplication, but needed to fix `isDrawerClosed(wrapper.findNavigation())` test util to make my tests in this PR pass